appdatabase/migrationsprevnodecfg/bindata.go

```diff
@@ -136,7 +136,7 @@ func _0001_appDownSql() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "0001_app.down.sql", size: 356, mode: os.FileMode(0644), modTime: time.Unix(1707305567, 0)}
+	info := bindataFileInfo{name: "0001_app.down.sql", size: 356, mode: os.FileMode(0644), modTime: time.Unix(1694703774, 0)}
```

Comment:
You can exclude unneeded binadata's from commit, for your PR only `protocol` make sense